ivy/container/linear_algebra.py

```diff
@@ -1084,6 +1119,39 @@ def vector_norm(
         *,
         out: Optional[ivy.Container] = None,
     ) -> ivy.Container:
+        """Compute vector p-norm for each array in the container.
```

Comment:
these docstrings are wrong, please re-read the **classes** section [here](https://github.com/unifyai/ivy/blob/e7b3211dbe1d24ff651155d7899546a7c4dc30f7/docs/partial_source/deep_dive/12_docstrings.rst)

Reply:
Yeah, you are right! I copied the legacy docstring in base.py.
I have updated it now :D